src/VCAuthn/appsettings.json

```diff
@@ -14,7 +14,19 @@
     },
     "CertificateFilename": "secret.rsa",
     "PublicOrigin": "http://localhost:5000",
-    "AccessTokenLifeTimeSeconds": 172800
+    "AccessTokenLifeTimeSeconds": 172800,
+    "Clients": [
+      {
```

Comment:
Is this a demo client? Do all clients need to be supplied here or is there a root client and API for managing them? What happens when this is changed between runs? I see it checks using id, so once run, no update of client details in here will be configured in db?

Reply:
It's a configuration for a test keycloak client.
There is no root client, neither there is an api to manage them. If more clients need to be added, application needs to be re-started.
I've changed it to update existing clients, that should be better.